docs/api/python/function.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: docu
+title: Python Function API
+selected: Client APIs
+---
+
+You can create a DuckDB function out of a python function so it can be used in SQL queries.
+Just like regular [functions](../../sql/functions/overview.md) they need to have a name, a return type and parameter types.
+
+### Creating Functions
+
+The `create_function` method is used to add a function, the parameters to this method are:  
+
+`name: str`  
```

Comment:
This section can probably just be a link to the API reference instead?

Reply:
I don't think that's descriptive enough.
https://duckdb.org/docs/api/python/reference/#duckdb.DuckDBPyConnection.create_function